explorer/src/queries/search.ts

```diff
@@ -23,20 +43,13 @@ const searchBuilder = (searchQuery?: string): SelectQueryBuilder<JobRun> => {
   let query = getRepository(JobRun).createQueryBuilder('job_run')
 
   if (searchQuery != null) {
-    const searchTokens = searchQuery.split(/\s+/)
-    const normalizedSearchTokens = searchTokens.map(normalizeSearchToken)
+    let searchTokens = searchQuery.split(/\s+/)
```

Comment:
It'd be nice if these two lines were put into their own function to make it a bit clearer, then searchTokens can be `const` as well for the duration of this function.